qiskit/providers/ibmq/ibmqbackend.py

```diff
@@ -78,19 +78,21 @@ def run(self, qobj):
 
         return job
 
-    def properties(self, refresh=False):
+    #TODO: Update docstring for db_filter
+    def properties(self, refresh=False, db_filter=None):
```

Comment:
For this particular function, can you try to be a bit more restrictive with the new parameter, and instead of allowing a filter lean on a more simple type (the one that first comes to mind is ... a datetime or similar)?

The reason is that this one is mainly an user-facing function (as opposed to the rest of functions changed, which are more "internal") - and actually the one that will define the "user API" part of the issue. In general it is about balancing the different forces, and for user-facing functions a good thumb rule is to tend to simplicity: in this case, and considering the requirement was to give the user the "ability to retrieve an older calibration" and the filter for that use case will always be the same with the only variable part being the timestamp, it makes sense to avoid having the user worry about the specific format of the filter (at least, until requirements change :grin:  ).

Reply:
Yeah, that makes more sense! I'm changing the parameter into a datetime, because it might be the most common and simple type.

Initially, I tried to replicate the way job filtering was done. But, I agree. This would make the filtering much more user friendly since it removes the complexity of the filtering format. Thanks for the tips @diego-plan9! 👍. I'll keep that in mind going forward as well.